.eslintrc.js

```diff
@@ -92,7 +92,7 @@ module.exports = {
         "no-trailing-spaces": "error",
         "no-undef-init": "error",
         "no-unsafe-finally": "error",
-        "no-unused-expressions": "error",
+        "no-unused-expressions": "off",
```

Comment:
is this change really needed?

Reply:
Yes, things like `getStatusBar()?.updateBuildPresetButton();` were being failed during build, but that seems unreasonable, when this is a good way to succinctly write this code.

Thanks.